api/utils/keys/piv/yubikey.go

```diff
@@ -692,3 +736,32 @@ func SelfSignedMetadataCertificate(subject pkix.Name) (*x509.Certificate, error)
 	}
 	return cert, nil
 }
+
+func isTeleportMetadataCertificate(cert *x509.Certificate) bool {
+	return len(cert.Subject.Organization) > 0 && cert.Subject.Organization[0] == certOrgName
+}
+
+func nonTeleportCertificateMessage(slot piv.Slot, cert *x509.Certificate) string {
+	// Gather a small list of user-readable x509 certificate fields to display to the user.
+	sum := sha256.Sum256(cert.Raw)
+	fingerPrint := hex.EncodeToString(sum[:])
+	return fmt.Sprintf(`Certificate in YubiKey PIV slot %q is not a Teleport client cert:
+Slot %s:
+	Algorithm:		%v
+	Subject DN:		%v
+	Issuer DN:		%v
+	Serial:			  %v
```

Comment:
```suggestion
	Serial:			%v
```
whitespace looks off here

Reply:
Ah it looked off in my editor, but I guess it was correct before